test/integration/generated/deviceprovisioningservice/src/models/parameters.ts

```diff
@@ -275,7 +275,7 @@ export const asyncinfo: OperationQueryParameter = {
 };
 
 export const argumentsParam: OperationParameter = {
-  parameterPath: "argumentsParam",
+  parameterPath: "arguments",
```

Comment:
This needs to be called `argumentsParam`
![image](https://user-images.githubusercontent.com/21109913/159977124-61f79c6e-e4b1-49e2-986f-ec245672c595.png)

Reply:
changed back